examples/SDL2/main.cpp

```diff
@@ -0,0 +1,15 @@
+#include<iostream>
+#include<SDL/SDL.h>
```

Comment:
I don't understand, `#include <SDL2/SDL.h>` breaks and `#include <SDL/SDL.h` works fine for me, can I take a look at your test perhaps?